components/org.wso2.carbon.identity.api.server.secret.management/org.wso2.carbon.identity.api.server.secret.management.common/src/main/java/org/wso2/carbon/identity/api/server/secret/management/common/SecretManagementConstants.java

```diff
@@ -36,14 +35,19 @@
     public enum ErrorMessage {
 
         // Client errors 600xx.
+        ERROR_CODE_CONFLICT_SECRET("00006", "Secret already exists.", "There exists a " +
```

Comment:
won't these error codes need to be started with 600 ?

And can order them in the error code order for better readability?

Reply:
This error is getting from the framework.
 https://github.com/wso2/carbon-identity-framework/blob/4745401388cab18c7513a5802ed64670abf1b2b3/components/secret-mgt/org.wso2.carbon.identity.secret.mgt.core/src/main/java/org/wso2/carbon/identity/secret/mgt/core/constant/SecretConstants.java#L45

Reply:
resolved. reused the code defined in framework